docs/advanced/topology-updater-commandline-reference.md

```diff
@@ -0,0 +1,190 @@
+---
+title: "Topology Updater Cmdline Reference"
+layout: default
+sort: 5
+---
+
+# NFD-Topology-Updater Commandline Flags
+{: .no_toc }
+
+## Table of Contents
+{: .no_toc .text-delta }
+
+1. TOC
+{:toc}
+
+---
+
+To quickly view available command line flags execute `nfd-topology-updater --help`.
+In a docker container:
+
+```bash
+docker run gcr.io/k8s-staging-nfd/node-feature-discovery:master nfd-topology-updater --help
+```
+
+### -h, --help
+
+Print usage and exit.
+
+### --version
```

Comment:
nit: drop the double dashes from the flags here i.e. just `-version` instead of `--version`. Makes it in line with the other command line references and also how the flags package presents them.

Reply:
Ah, this applies to all other flags, too 😄